<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tuan1</class>
 <widget class="QWidget" name="Tuan1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tuan1</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lựa chọn bài tập</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai9">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai11">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 11</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai12">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 12</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
